Desktop/widgets/comboboxbase.cpp

```diff
@@ -116,34 +120,40 @@ void ComboBoxBase::termsChangedHandler()
 {
 	std::vector<std::string> options;
 	const Terms& terms = _model->terms();
-	int counter = 0;
 	int index = -1;
-	for (const Term& term : terms)
+
+	if (initialized())
 	{
-		QString label = term.asQString();
-		QString value = _model->getValue(label);
-		options.push_back(value.toStdString());
-		if (value == _currentValue)
-			index = counter;
-		counter++;
+		int counter = 0;
```

Comment:
Why not just do `for(size_t counter=0; counter < terms.size(); counter++)`  then?

Reply:
OK